build.sh

```diff
@@ -240,7 +284,7 @@ __ProjectRoot="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # Use uname to determine what the CPU is.
 CPUName=$(uname -p)
 # Some Linux platforms report unknown for platform, but the arch for machine.
-if [ $CPUName = "unknown" ]; then
+if [ $CPUName == "unknown" ]; then
```

Comment:
Shell portability issue. This will fail on NetBSD. `==` is extension in `test`(1).